.env.sample

```diff
@@ -28,4 +28,79 @@ KEYCLOAK_PUBLIC_KEY_PATH = "keycloak-public-keys"
 
 DISABLE_LEARNER_SERVICE_ON_OFF = "ON"                                          // Disable learner service check
 
-FORM_SERVICE_URL = "http://player:3000"                                         // Base url for form search
\ No newline at end of file
+FORM_SERVICE_URL = "http://player:3000"                                         // Base url for form search
+
+
+# Redis configuration
+
+REDIS_URL = redis://localhost:6379                              // Redis connection url
```

Comment:
@swetha-aggidevara Do we need redis for migration? If not can we keep only the code for migration